include/envoy/upstream/load_balancer_type.h

```diff
@@ -17,6 +19,11 @@ namespace Upstream {
  */
 enum class LoadBalancerType { RoundRobin, LeastRequest, Random, RingHash, OriginalDst, Maglev };
 
+typedef std::pair<
```

Comment:
Any specific reason for using std::pair instead of struct here? Could be more readable to use descriptive names instead of first and second.

Reply:
@yanavlasov thanks for hint, will switch to struct instead

Reply:
switched to struct instead